asn1acn.pro

```diff
@@ -162,15 +168,28 @@ HEADERS += \
     modeltree.h \
     asn1acnjsextension.h \
     asn1sccserviceprovider.h \
-    parseddocumentbuilder.h \
+    asn1sccparseddocumentbuilder.h \
     documentsourceinfo.h \
     projectcontenthandler.h \
     indenter.h \
     tr.h \
     linkcreator.h \
     tools.h \
     overviewindexupdater.h \
-    structuresviewindexupdater.h
+    structuresviewindexupdater.h \
+    parsingserviceprovider.h \
+    asn1sccserviceproviderstub.h \
```

Comment:
imho stubs etc should go into `tests` subdir and be built only when "WITH_TESTS" is true